### ISSUE TITLE
[Feature Request]: Log thread ID in built-in loggers

### ISSUE THREAD

**Issue by ekzhu** [microsoft/autogen#2656]
### Is your feature request related to a problem? Please describe.

Currently the built-in loggers do not log the current thread ID. This makes it difficult to differentiate between logs from different threads. 

> will there be a way to differentiate between the different threads chat completions? Id need to be able to query the sqlite db for all chat completions of a single thread
by @Nathan-Intergral 

### Describe the solution you'd like

In SQLiteLogger and FileLogger (#2596), add thread ID to each logged event. Obtain thread ID through: 

```python
import threading
current_thread_id = threading.get_native_id()
print(f"Current thread ID (native): {current_thread_id}")
```

### Additional context

#2617

**Comment by jtoy** [comment]
This is merged, can we close this ticket.

**Comment by ekzhu** [comment]
it is not yet in the SQLIteLogger.

**Comment by lucaswh** [comment]
If I understand correctly, this threadId logging functionality is available in the FileLogger but not the SQLiteLogger. Is that right, @ekzhu ?

**Comment by ekzhu** [comment]
> If I understand correctly, this threadId logging functionality is available in the FileLogger but not the SQLiteLogger. Is that right, @ekzhu ?

Yes

**Comment by Nathan-Intergral** [comment]
Hi, i have had to work on other projects for a while but i am back working with autogen again. Was this ever progressed?

**Comment by ekzhu** [comment]
@Nathan-Intergral It is available now in `autogen-agentchat~=0.2`

**Comment by ekzhu** [comment]
Closing this for now as team is focusing on v0.4 release. 

You are welcome to open a new feature request issue under for the new logger handlers -- currently SQLite log handler is not supported. We can discuss whether this should be included in the core library, as an extension module, or a community package.
